AdobeStockImageAdminUi/view/adminhtml/web/template/grid/column/image-preview.html

```diff
@@ -8,13 +8,13 @@
 <div class="masonry-image-preview" if="$col.isVisible($row())" data-image-preview ko-style="$col.getStyles($row())">
     <div class="container">
         <div class="action-buttons">
-            <button class="action-previous" type="button" data-bind="click: function () { $col.prev($row()); }">
+            <button class="action-previous" type="button" data-bind="click: function () { $col.prev($row()); $col.hideAllKeywords($row()); }">
                 <span translate="'Previous'"/>
```

Comment:
Hiding keywords on switching between images is a part of view layer logic, it should not be tiered to the template. Please extend prev, next and hide methods in the component.

Reply:
Ok